doc/frontend.md

```diff
@@ -7,14 +7,12 @@ interface for superusers to manage other users.
 
 ## Contributing
 
-### GOV.UK Frontend
+### Design System
 
-The Frontend app makes use of the
-[GOV.UK Frontend](https://design-system.service.gov.uk/get-started/production/)
-design system to markup and style the web interface.
+The Frontend app makes use of the [GOV.UK Frontend Design System](https://design-system.service.gov.uk/get-started/production/) and the [Ministry of Justice Design System](https://github.com/ministryofjustice/moj-design-system)  to markup and style the web interface.
```

Comment:
It's called GOV.UK Design System - that's the overall guidance, the bit we use (the npm package) is called GOV.UK Frontend.